### PR TITLE
Adding the missing pins(33,34) and SPI pins

### PR DESCRIPTION
**Pull request by tirtharajsinha**

### CODE REVIEW

File: ports/espressif/boards/waveshare_esp32_s3_pico/pins.c [hunk 1]
```diff
@@ -25,6 +25,8 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_IO19), MP_ROM_PTR(&pin_GPIO19) },
     { MP_ROM_QSTR(MP_QSTR_IO20), MP_ROM_PTR(&pin_GPIO20) },
     { MP_ROM_QSTR(MP_QSTR_IO21), MP_ROM_PTR(&pin_GPIO21) },
+    { MP_ROM_QSTR(MP_QSTR_IO33), MP_ROM_PTR(&pin_GPIO33) },
+    { MP_ROM_QSTR(MP_QSTR_IO34), MP_ROM_PTR(&pin_GPIO34) },
```
Comment (dhalbert) [inline ports/espressif/boards/waveshare_esp32_s3_pico/pins.c:29]: These are not broken out on this particular board, or am I missing something? They are labeled in the schematic, but the module pins are not connected to anything.

**Review by dhalbert — DISMISSED** [review]

**Comment by michael-ring** [comment]
fyi I tested with my own version, the two pins work as expected, at least as standard GPIO pins

File: ports/espressif/boards/waveshare_esp32_s3_pico/pins.c (hunk 1 above)
Comment (dhalbert) [reply ports/espressif/boards/waveshare_esp32_s3_pico/pins.c:29]: Sorry, never mind. I was confused with the GPIO43 and GPIO44 pins.

**Review by dhalbert — APPROVED** [review]
Thanks! Sorry for my mixup on the pins in my initial review.

**Comment by tirtharajsinha** [comment]
thanks

**Pull request merged**